### DISCUSSION TITLE
vitis安裝問題

### DISCUSSION THREAD

**Discussion by fefefw** [bol-edu/HLS-SOC-Discussions#230] [General]
照指示下載完打開後出現下面這個問題  請問該如何解決 謝謝!
![image](https://github.com/user-attachments/assets/f833e8be-c73f-4628-a095-b9fa8ea84b47)
![image](https://github.com/user-attachments/assets/8cb838fd-da47-4ad4-9108-0fbe5554cbe6)

**Comment by asdshawn** [comment]
你用什麼作業系統？看起來不像是Windows阿
